roles/common/files/uug_ansible_wrapper.py

```diff
@@ -578,12 +574,101 @@ def get_distro_release_name():
     return release
 
 
+def validate_branch_settings(self):
+    """
+    Warns the user of an error in the settings of the VM configuration. 
+    :returns: a boolean indicating if the system should return or continue
+    """
+    
+    system_version = get_distro_release_name()
+    chosen_branch = USER_CONFIG['git_branch']
+    chosen_remote = USER_CONFIG['git_url']
+    branch_mismatch = system_version != chosen_branch
+    looks_minty = re.compile(r"[a-z]+a").fullmatch(system_version)
+    
+    output = subprocess.run(
+        ["/usr/bin/git", "ls-remote", '--heads', USER_CONFIG['git_url']],
+        stdout=subprocess.PIPE
+    )
+    
+    ls_remote = output.stdout.decode("utf-8")
+    
+    header = None
+    warning_prompt = None
+    
+    system_exists = system_version in ls_remote
+    chosen_exists = chosen_branch in ls_remote
+        
+    if chosen_remote == DEFAULT_GIT_REMOTE:
```

Comment:
You can remove a whole indentation level by inverting the condition.

```python
if chosen_remote != DEFAULT_GIT_REMOTE:
    return True
```

And then unindent the rest of the if statements.